corehq/apps/linked_domain/migrations/0005_migrate_linked_app_toggle.py

```diff
@@ -37,6 +14,4 @@ class Migration(migrations.Migration):
         ('linked_domain', '0004_domainlinkhistory'),
     ]
 
-    operations = [
-        migrations.RunPython(_migrate_linked_apps_toggle, noop)
-    ]
+    operations = []
```

Comment:
I think removing migrations like this is not safe. What if someone still needs to run it, but has not yet?

Reply:
I agree that modifying existing migrations is generally not safe. However I believe it's safe in this instance as it was migrating from an old toggle that no longer exists to a new toggle. This toggle hasn't existed for 10 months at this point. I think this is somewhat similar to the time we squashed accounting migrations which had a lot of custom migration code in them.

With that said, I would like for us to have a standard for when we decide to do this sort of thing. 10 months seems like quite a reasonable time to me.

Reply:
Fair point. I got bitten by not running migrations and I can no longer migrate my dev environment. So I write automated tests rather than testing manually in a local dev environment 👅 

I'm not sure what the best time horizon is for modifying old migrations. A year seems easier to remember than 10 months, but I'm not sure I care that much. We are not testing applying migrations to old, unmigrated data anyway, so it's hard to catch when we violate this rule or even more subtly, when we make breaking changes to code imported by but outside of migration files.